apps/ui/src/components/forms/ContactForm.tsx

```diff
@@ -0,0 +1,108 @@
+"use client"
+
+import React, { useState } from "react"
+import Link from "next/link"
+import { zodResolver } from "@hookform/resolvers/zod"
+import { Attribute } from "@repo/strapi"
+import { useTranslations } from "next-intl"
+import { useForm } from "react-hook-form"
+import * as z from "zod"
+
+import { AppField } from "@/components/forms/AppField"
+import { AppForm } from "@/components/forms/AppForm"
+import { AppTextArea } from "@/components/forms/AppTextArea"
+import { Button } from "@/components/ui/button"
+
+export function ContactForm({
+  gdpr,
+}: Readonly<{
+  title?: string | undefined
+  description?: string | undefined
+  gdpr?: Attribute.GetValues<"shared.link">
```

Comment:
Lets change type of `gdpr` prop to be more generic:

```tsx
gdpr?: {href?: string, label?: string}
```

With that we can use `ContactForm` anywhere without connection to Strapi component